src/Sylius/Behat/Context/Domain/ProductContext.php

```diff
@@ -28,20 +31,36 @@
     private $sharedStorage;
 
     /**
+     * @var RepositoryInterface
+     */
+    private $productRepository;
+
+    /**
      * @var ProductVariantRepositoryInterface
      */
-    private $productVariantRepository;
+    private $variantRepository;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $reviewRepository;
 
     /**
      * @param SharedStorageInterface $sharedStorage
-     * @param ProductVariantRepositoryInterface $productVariantRepository
+     * @param RepositoryInterface $productRepository
+     * @param ProductVariantRepositoryInterface $variantRepository
+     * @param RepositoryInterface $reviewRepository
      */
     public function __construct(
         SharedStorageInterface $sharedStorage,
-        ProductVariantRepositoryInterface $productVariantRepository
+        RepositoryInterface $productRepository,
+        ProductVariantRepositoryInterface $variantRepository,
+        RepositoryInterface $reviewRepository
     ) {
         $this->sharedStorage = $sharedStorage;
-        $this->productVariantRepository = $productVariantRepository;
+        $this->productRepository = $productRepository;
+        $this->variantRepository = $variantRepository;
```

Comment:
Why renaming `productVariantRepository` to `variantRepository`? We still use `ProductVariantRepositoryInterface`.